release-notes/app-catalog/app-platform-announcement.md

```diff
@@ -0,0 +1,51 @@
+# Announcing Giant Swarm App Platform
+We are pleased to announce that the Giant Swarm App Platform is generally available.
+
+In the past few months, you may have noticed that we have been working on this. We even answered some questions we received as a result of the preview. This has helped us shape the platform and bring it to where it is today.
+
+There are two ways to benefit:
```

Comment:
The benefits here are a summary of the catalogs below. But I think these are not the benefits of the app platform itself, i.e. installing an app is not a benefit. The second point might be more of a benefit. The benefits are maybe clearer in the 3 points we have on the current app catalog page of our website + the benefit of automating the mgmt of "(sets of) apps on fleets of cluster".

cc @Oshratn

Reply:
Puja agrees w the suggestion below, so this is resolved.